src/api/search/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@senecacdot/search-service",
+  "private": true,
+  "version": "1.0.0",
+  "description": "A service for optimizing images",
+  "scripts": {
+    "dev": "nodemon server.js | pino-pretty -c -t",
+    "start": "node server.js",
+    "test": "jest test.js"
```

Comment:
Test config and execution all happens from the root now.  Setup your tests like we do in image, feed-discovery, posts, etc. https://github.com/Seneca-CDOT/telescope/blob/master/src/api/image/jest.config.js